packages/react-app-scaffolder/app/templates/no-redux/src/core/private-route-wrapper.tsx

```diff
@@ -38,15 +38,19 @@ export const PrivateRouteWrapper: React.FunctionComponent<PrivateRouteWrapperPro
   return (
     <AppNavContainer>
       <Menu />
-      <Suspense
-        fallback={
-          <Section>
-            <Loader />
-          </Section>
-        }
-      >
-        {children}
-      </Suspense>
+      <FlexContainerBasic flexColumn isScrollable>
+        <FlexContainerResponsive hasPadding flexColumn>
+          <Suspense
+            fallback={
+              <Section>
+                <Loader />
+              </Section>
+            }
+          >
+            {children}
```

Comment:
Suggest to move `<ErrorBoundary>{children}</ErrorBoundary>` to here and can reuse everywhere